### ISSUE TITLE
hbro-1.4.0.0 does not compile

### ISSUE THREAD

**Issue by peti** [k0ral/hbro#15]
http://hydra.cryp.to/build/1054765/nixlog/1/raw:

~~~~
[20 of 32] Compiling Hbro.Keys        ( library/Hbro/Keys.hs, dist/build/Hbro/Keys.o )

library/Hbro/Keys.hs:52:10: Warning:
    Orphan instance: instance Describable Gdk.Modifier
    To avoid this
        move the instance declaration to the module of the class or of the type, or
        wrap the type with a newtype and declare the instance on the new type.

library/Hbro/Keys.hs:58:1: Warning:
    Orphan instance: instance Ord Gdk.Modifier
    To avoid this
        move the instance declaration to the module of the class or of the type, or
        wrap the type with a newtype and declare the instance on the new type.
[21 of 32] Compiling Hbro.Clipboard   ( library/Hbro/Clipboard.hs, dist/build/Hbro/Clipboard.o )
[22 of 32] Compiling System.Process.Extended ( library/System/Process/Extended.hs, dist/build/System/Process/Extended.o )
[23 of 32] Compiling Hbro.Gui.NotificationBar ( library/Hbro/Gui/NotificationBar.hs, dist/build/Hbro/Gui/NotificationBar.o )

library/Hbro/Gui/NotificationBar.hs:66:1: Warning:
    Defined but not used: ‘write’
[24 of 32] Compiling Hbro.WebView.Signals ( library/Hbro/WebView/Signals.hs, dist/build/Hbro/WebView/Signals.o )
[25 of 32] Compiling System.Glib.Attributes.Extended ( library/System/Glib/Attributes/Extended.hs, dist/build/System/Glib/Attributes/Extended.o )
[26 of 32] Compiling Hbro.Gui.MainView ( library/Hbro/Gui/MainView.hs, dist/build/Hbro/Gui/MainView.o )

library/Hbro/Gui/MainView.hs:264:20:
    Ambiguous occurrence ‘scroll’
    It could refer to either ‘Hbro.Gui.MainView.scroll’,
                             defined at library/Hbro/Gui/MainView.hs:269:1
                          or ‘Graphics.UI.Gtk.WebKit.DOM.Document.scroll’,
                             imported from ‘Graphics.UI.Gtk.WebKit.DOM.Document’ at library/Hbro/Gui/MainView.hs:62:1-52

library/Hbro/Gui/MainView.hs:265:20:
    Ambiguous occurrence ‘scroll’
    It could refer to either ‘Hbro.Gui.MainView.scroll’,
                             defined at library/Hbro/Gui/MainView.hs:269:1
                          or ‘Graphics.UI.Gtk.WebKit.DOM.Document.scroll’,
                             imported from ‘Graphics.UI.Gtk.WebKit.DOM.Document’ at library/Hbro/Gui/MainView.hs:62:1-52
~~~~

**Comment by k0ral** [comment]
Thanks for reporting.
I'll have a look as soon as possible.

**Comment by k0ral** [comment]
Fixed in #16 .

**Comment by peti** [comment]
Will you release a new version?

**Comment by k0ral** [comment]
I've just released 1.4.0.1 with this fix, sorry for the delay.
